### DISCUSSION TITLE
Ruined Fooocus and Fooocus both installed OK?

### DISCUSSION THREAD

**Discussion by PeterMcLennan** [runew0lf/RuinedFooocus#123] [Q&A] [answered]
My Fooocus install is working perfectly.  Can I install Ruined Fooocus on the same drive without endangering my current install?

**Comment by mald55** [comment, marked as answer]
You should be fine in case you haven't done so already, the vanilla Fooocus and MRE installed at the same time.  You should be even able to run both at the same time too.

**Reply by PeterMcLennan** [reply]
Thanks, mald55!  I look forward to trying it.
